microsoftgraph/docs-conceptual/app-only.md

```diff
@@ -2,7 +2,6 @@
 title: "Use app-only authentication with the Microsoft Graph PowerShell SDK"
 description: "Learn how to use app-only authentication to enable non-interactive scenarios with the Microsoft Graph PowerShell SDK."
 
-ms.localizationpriority: medium
 author: jasonjoh
```

Comment:
@msewaweru Please add the manager field here. It's not so much direct manager as business owner, M1 PoC for the article in terms of reporting, escalation, etc. Can put me here and for the others with Jason as author. Thank you.